build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -520,6 +520,49 @@ message Platform {
   // The server MAY use the `value` of one or more properties to determine how
   // it sets up the execution environment, such as by making specific system
   // files available to the worker.
+  //
+  // The following standard `name`s are defined:
+  //
+  // * `OS`: This describes the operating system. Only a single value is allowed
```

Comment:
I'd vote to lowercase everything - keys ("OS" -> "os"), values, etc.

Reply:
That's fine with me. Will change it unless there are objections.

Reply:
My preference is lower-dash-separated (I don't know if there's a formal name for that style). Of course, for "os" and "isa" there are no dashes. :)

Reply:
For worker attributes specified via the REAPI platform variables, RBE has been Pascal casing for "standard" attributes and camel casing for non-standard attributes, with initialisms maintaining fixed casing throughout (i.e., all lower-case if they appear at the beginning of a non-standard attribute name, all upper-case otherwise).

For example, we have been using `OSFamily` (note that "OS" is all-caps) and `Pool` as "standard" attributes and `gceMachineType` (note that "gce" is all lowercase) as a non-standard attribute. My preference would be to maintain this convention.

Reply:
I'd be fine either way for the keys, all lower case or Pascal case (the abbreviations `OS` and `ISA` would remain all-caps, I presume). However, for the values Pascal case would not be ideal, in my opinion, especially for ISA values I'd prefer all lower case to keep it clear and simple. Does this work for you?

Reply:
I've kept the keys Pascal case in line with RBE but I also wouldn't have any objections to change this to lower-dash-separated.

The `ISA` values are lower-dash-separated and I think that's sensible, especially as there isn't a clear Pascal case equivalent for all values.

I've changed the `OSFamily` values to lower-dash-separated to be in line with the `ISA` values. However, Pascal case would work ok for `OSFamily` values as well.

Is this acceptable to everyone?